tripzone/src/components/Button/Button.js

```diff
@@ -0,0 +1,29 @@
+import "./Button.scss";
+import PropTypes from 'prop-types';
+import classnames from "classnames";
+import { AppConstants } from "../../constants/App.constants";
+
+const { ATTRIBUTES } = AppConstants.COMMONS.BUTTON;
```

Comment:
i think these are for size attributes and not just attributes, could you try giving relevant names